src/main/kotlin/com/dansplugins/factionsystem/law/JooqMfLawRepository.kt

```diff
@@ -38,15 +48,56 @@ class JooqMfLawRepository(
     }
 
     override fun delete(id: MfLawId) {
+        delete(getLaw(id)!!)
```

Comment:
Remember that these operations can occur concurrently due to the async blocks - getLaw may not exist so asserting it does may cause errors.